clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/model/Manifest.java

```diff
@@ -46,19 +46,19 @@
   @NonnullByDefault
   @ToString
   class Status {
-    private @Nullable Condition stable = Condition.withState(true);
+    private Condition stable = Condition.withState(true);
     private Condition paused = Condition.withState(false);
     private Condition available = Condition.withState(true);
-    private @Nullable Condition failed = Condition.withState(false);
+    private Condition failed = Condition.withState(false);
 
     public static Status defaultStatus() {
       return new Status();
     }
 
-    public Status unknown() {
```

Comment:
The removal of `unknown` is the only change in this PR that could break custom handlers (as the other changes are all implementation-specific changes to built-in handlers).  I think the benefits of null-safety and making it easier to understand outweigh this; anyone who had called `unknown()` to null out fields on the status will need to pick a new appropriate mutation; likely `Status.unstable()` with an appropriate message about why it's unstable.  I opted to remove the method rather than just make it do that automatically because I think it's worth clients figuring out why they were calling `unknown()` and instead doing something more appropriate rather than changing the meaning of `unknown()` but leaving their code compiling.